src/components/vice/admin/filter/sections.js

```diff
@@ -57,6 +57,7 @@ const FilterSection = ({
                     />
 
                     <TextField
+                        variant="standard"
```

Comment:
I always thought the VICE admin text and select fields would look better as `outlined` variants.